doc/src/build/move/manifest.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Move.toml
+---
+
+Every Move package has a `Move.toml` file as a manifest for the package. The Sui Move compiler creates the `Move.toml` file for you when you create a new package, but you might need to update the file throughout the lifespan of your package.    
```

Comment:
@amnn Is 'Sui Move compiler` the correct actor? I'll update all references if it isn't.

Reply:
I would just call it the "Sui CLI" or whatever we use to refer to that entity, rather than the compiler specifically.